hw/ip/pwrmgr/data/pwrmgr_sec_cm_testplan.hjson

```diff
@@ -25,81 +25,187 @@
+      desc: '''Verify the countermeasure(s) ROM_CTRL.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Use comprehensive stimulus - reset and wakeup -
+              as background traffic to ensure this counter measure
+              is valid for various states of fast and slow fsm.
+            - Drive rom_ctrl_i with mixed valid and invalid values.
+
+            **Check**:
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.rom_ctrl_i
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_rom_ctrl_intersig_mubi"]
     }
     {
       name: sec_cm_rstmgr_intersig_mubi
-      desc: "Verify the countermeasure(s) RSTMGR.INTERSIG.MUBI."
+      desc: '''Verify the countermeasure(s) RSTMGR.INTERSIG.MUBI.
+
+            **Stimulus**:
+            - Drive tb.dut.sw_rst_req_i with mixed valid and invalid values
+
+            **Check**:
+            - See sw rst only happens when dut gets valid value.
+            - Collect coverage by binding cip_mubi_cov_if to
+              tb.dut.sw_rst_req_i
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm_rstmgr_intersig_mubi"]
     }
     {
       name: sec_cm_esc_rx_clk_bkgn_chk
-      desc: "Verify the countermeasure(s) ESC_RX.CLK.BKGN_CHK."
+      desc: '''Verify the countermeasure(s) ESC_RX.CLK.BKGN_CHK.
+
+            **Stimulus**:
+            - At FastPwrStateActive state, create escalation clock
+              or reset failure by stopping clock or asserting reset.
+
+            **Check**:
+            - Expecting fatal alert event. After alert agent process
+              the alert by asserting escalation reset, see if dut
+              is back to normal operation state.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_esc_clk_rst_malfunc"]
     }
     {
       name: sec_cm_esc_rx_clk_local_esc
-      desc: "Verify the countermeasure(s) ESC_RX.CLK.LOCAL_ESC."
+      desc: '''Verify the countermeasure(s) ESC_RX.CLK.LOCAL_ESC.
+
+            This is triggered by common cm primitives (SecCmPrimCount).
+            (https://github.com/lowRISC/opentitan/blob/master
+            /hw/dv/sv/cip_lib/doc/index.md#security-verification
+            -for-common-countermeasure-primitives)
+
+            **Check**:
+            - Detect fast state transition to FastPwrStateResetPrep.
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm"]
     }
     {
       name: sec_cm_fsm_sparse
-      desc: "Verify the countermeasure(s) FSM.SPARSE."
+      desc: '''Verify the countermeasure(s) FSM.SPARSE.
+            This is triggered by common cm primitives (SecCmPrimSparseFsmFlop).
+            (https://github.com/lowRISC/opentitan/blob/master
+            /hw/dv/sv/cip_lib/doc/index.md#security-verification
+            -for-common-countermeasure-primitives)
+            '''
       milestone: V2S
-      tests: []
+      tests: ["pwrmgr_sec_cm"]
```

Comment:
in addition to making sure the alert fires as part of the prim, `pwrmgr` also has some extra behavior.
What is the best way to check that?

Reply:
In my interpretation, sec_cm_fsm_sparse and sec_cm_fsm_local_esc are correlated. see line 134 of this file.
So upon triggering the event, test (a single test for both) will go through state machine and expected output check